xeeg_hed_score/sub-eegArtifactTUH/ses-eeg01/eeg/sub-eegArtifactTUH_ses-eeg01_task-rest_run-000_events.tsv

```diff
@@ -0,0 +1,15 @@
+onset	duration	annotation_type	channel
+36.2312	39.807	eyem	FP1-F7
```

Comment:
channel column should match channel name throughout

Reply:
See [channels.tsv](https://github.com/bids-standard/bids-examples/blob/b8599a75164f015a40b61cb4347c47ab08a878ac/xeeg_hed_score/sub-eegArtifactTUH/ses-eeg01/eeg/sub-eegArtifactTUH_ses-eeg01_channels.tsv), how should channels with 2 different references like 'FP1-F7' and 'FP1-F3' be handled?

Reply:
In my mind, you should create two channels in the data, with two distinct names -- hence two rows in `channels.tsv`.

 But I believe @robertoostenveld may have a different opinion on this.

Reply:
Updated _channels.tsv to accommodate 2 different references.